src/fluvio-socket/.github/workflows/ci.yaml

```diff
@@ -0,0 +1,65 @@
+name: CI
```

Comment:
Do we wanna remove this?

Reply:
Yes, there are some things in here that are duplicate infrastructure, but I want to make sure that anything that was unique in fluvio-socket or fluvio-protocol gets merged into the fluvio infra before deleting it